fusequery/benchmarks/nyctaxi/src/bin/nyctaxi.rs

```diff
@@ -14,7 +14,7 @@ use common_planners::CreateTablePlan;
 use common_planners::TableEngineType;
 use common_planners::TableOptions;
 use fuse_query::interpreters::InterpreterFactory;
-use fuse_query::optimizers::Optimizer;
+use fuse_query::optimizers::Optimizers;
```

Comment:
What is the reason for this change?

Reply:
There is a naming confliction between `trait IOptimizer` and `struct Optimizer` (before refactoring).

`struct Optimizer` holds a vector of IOptimizer (thus, Optimizer**s**), like this:
~~~
pub struct Optimizer {    optimizers: Vec<Box<dyn IOptimizer>> }
pub trait IOptimizer { ....}
~~~

To resolve the naming confliction, it seems to me that rename `struct Optimizer` to `struct Optimizers` and `trait IOptimizer`  to `trait Optimizer` is a reasonable approach. 

Any suggestions?

Reply:
It makes sense to rename Optimizer to Optimizers since the field is named Optimizers.

Reply:
Oh, forget to mention that field `optimizers`  is renamed to `inner` as well.